src/status_im/contexts/wallet/collectible/view.cljs

```diff
@@ -55,14 +64,72 @@
     :label               (i18n/label :t/about)
     :accessibility-label :about-tab}])
 
-(defn view
+(defn animated-header
+  [{:keys [scroll-amount title-opacity
+           type picture title description theme]}]
+  (let [blur-amount   (worklets/blur-amount scroll-amount)
+        layer-opacity (worklets/layer-opacity
+                       scroll-amount
+                       (colors/theme-colors "#ffffff00" "#00000000" theme)
```

Comment:
The colors have 100% opacity?
Why?
Why not just use transparent?

Reply:
Updated the code to use `transparent`. Thanks for pointing that out 😊